### PR TITLE
supports options in ElasticaAdapter

### PR DESCRIPTION
**Pull request by maarekj**
Add support options parameters in ElasticaAdapter

### CODE REVIEW

File: src/Pagerfanta/Adapter/ElasticaAdapter.php [hunk 1]
```diff
@@ -72,9 +78,11 @@ public function getResultSet()
      */
     public function getSlice($offset, $length)
     {
-        return $this->resultSet = $this->searchable->search($this->query, array(
+        $options = $this->options === null ? array() : $this->options;
```
Comment (stof) [inline src/Pagerfanta/Adapter/ElasticaAdapter.php]: couldn't we force options to always be an array in the constructor (defaulting to an empty array) ?

Comment (maarekj) [inline src/Pagerfanta/Adapter/ElasticaAdapter.php]: Good observation.

**Comment by sampart** [comment]
Thanks for contributing, @maarekj!  I'm really sorry it's been so long getting this merged.

**Comment by sampart** [comment]
This is now in release https://github.com/whiteoctober/Pagerfanta/releases/tag/v1.1.0

**Pull request merged**